circuit-lib/src/elgamal-babyjub/elgamal.ts

```diff
@@ -0,0 +1,444 @@
+import { AffinePoint } from "@noble/curves/abstract/curve";
+import { ExtPointType } from "@noble/curves/abstract/edwards";
+import { Scalar } from "ffjavascript";
```

Comment:
I would like to avoid using ff javascript. Can you get the functionality from a noble crypto library as well?

Reply:
It might be hard coded directly since I am just using it for a binary function1

Reply:
The `{ Scalar }` imported from **ffjavascript** is used for the function `formatPrivKeyForBabyJub()` -> Now the function is imported from `maci-crypto` so the `Scalar` import is omitted.